ipynb/examples/utils/testenv_example.ipynb

```diff
@@ -22,18 +38,86 @@
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2016-12-08 11:54:02,743 INFO    : root         : Using LISA logging configuration:\n",
+      "2016-12-08 11:54:02,744 INFO    : root         :   /home/vagrant/lisa/logging.conf\n"
+     ]
+    }
+   ],
    "source": [
-    "import json\n",
-    "import time\n",
-    "import os"
+    "from conf import LisaLogging\n",
+    "LisaLogging.setup()\n",
+    "# For debug information use:\n",
+    "# LisaLogging.setup(level=logging.DEBUG)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test environment setup"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Test environment setup"
+    "**Target configuration**:\n",
+    "\n",
+    "   - **platform** - the currently supported boards are:\n",
+    "        - linux - accessed via SSH connection\n",
+    "        - android - accessed via ADB connection\n",
+    "        - host - ran on local host\n",
+    "        \n",
+    "   - **board** - the currently supported boards are:\n",
+    "        - juno  - target is a JUNO board\n",
```

Comment:
This is no longer necessary. There is only "juno" with mainline hwmon, which was previously called juno2.